RedDeadOxygen/Assets/Scripts/Instances/Player.cs

```diff
@@ -32,10 +32,22 @@ public class Player : MonoBehaviour
     private Timer _speedTimer;
     private List<Ressource> _Ressources = new List<Ressource>();
 
+    private MovementCategory _movementCategory = MovementCategory.e_ForceBased;
+    private float _wallElasticity = 0f;
+
+    Vector2 movement, movementOrder;
```

Comment:
Multi-Line + name convention (be careful, variable reference should be changed too)
```suggestion
    Vector2 _movement; 
    Vector2 _movementOrder;
```